coordinator/src/routes/debug/convert.ts

```diff
@@ -25,7 +25,7 @@ export default [
       const payload = getPayload(request) as fhir.Resource
       const requestId = request.headers["nhsd-request-id"].toUpperCase()
       if (isBundle(payload)) {
-        const issues = bundleValidator.verifyBundle(payload)
+        const issues = bundleValidator.verifyBundle(payload, request.headers)
```

Comment:
Maybe have a verifyAuth helper method separately instead? Attaching to bundle validator feels a bit forced, i.e. auth isn't an issue with the bundle itself

Reply:
Also reading the implementation for the auth verification I think we'll need this for task endpoint too, as it is checking whether user/app auth is valid so we'll want those checks for task too if we want to support unattended and attended in future?

Reply:
thats true however the bundle validator is where we check whether the incoming bundle is part of the prescription flow or the dispense flow (and so requires different auth). in terms of task endpoints, everything atm is assumed app restricted unless it has the user auth verification on it. maybe could use some refactoring to make more clear?

Reply:
Yeah I think some refactoring here would help clear things up